gcloud-java-contrib/gcloud-java-nio/pom.xml

```diff
@@ -50,11 +50,6 @@
       <scope>provided</scope> <!-- to leave out of the all-deps jar -->
```

Comment:
If there are classes from this library that are needed in runtime than we can not specify it as provided as that means: compile time or included in the running environment (e.g. SDK extensions, container API,...). I am not very familiar with auto-service, but I think you do need
its classes in runtime.